reference/config_files/conandata.yml.rst

```diff
@@ -20,12 +20,20 @@ For example:
       "1.71.0":
         url: "https://dl.bintray.com/boostorg/release/1.71.0/source/boost_1_71_0.tar.bz2"
         sha256: "d73a8da01e8bf8c7eda40b4c84915071a8c8a0df4a6734537ddde4a8580524ee"
-    patches:
```

Comment:
Maybe it would be good to keep the patches too, and show in the recipe code how to apply them?